src/library/dao/trackdao.cpp

```diff
@@ -1022,146 +1049,131 @@ void TrackDAO::afterPurgingTracks(
 
 namespace {
 typedef bool (*TrackPopulatorFn)(const QSqlRecord& record,
-                                 const int column,
-                                 TrackPointer pTrack);
+        const int column,
+        TrackPointer pTrack);
 
-bool setTrackArtist(const QSqlRecord& record, const int column,
-                    TrackPointer pTrack) {
+bool setTrackArtist(const QSqlRecord& record, const int column, TrackPointer pTrack) {
```

Comment:
Accordeing to the iso cpp guide these functions should use a plain pointer because they do not store it. 

Not sure if this is a topic for this PR but it pops up during the mass reformat.

Reply:
Same here, unrelated